cfgov/cal/views.py

```diff
@@ -122,8 +181,16 @@ def display(request, pdf=False):
             'range_end':range_end,
         })
 
-        if pdf and PDFreactor:
-            return pdf_response(request, context)
+        if pdf:
+            template_name = 'about-us/the-bureau/leadership-calendar/print/index.html'
+            if PDFreactor:
+                return pdf_response(request, context)
+
+    elif form_is_vaild == False and pdf:
+        for error in form.errors.itervalues():
+            messages.error(request, str(error), extra_tags='leadership-calendar')
+
+        return HttpResponseRedirect('/the-bureau/leadership-calendar/')
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e